backend/backend/api/handlers/room.py

```diff
@@ -1,11 +1,11 @@
 from fastapi import status, Request
 from starlette.responses import JSONResponse
 
-from backend.errors.room import PlayerNotInRoom, PlayerIsNotLeader
+from backend.errors.room import PlayerNotInRoomError, PlayerIsNotLeaderError
 
 
 async def player_is_not_leader_handler(request: Request,
-                                       exc: PlayerIsNotLeader) -> JSONResponse:
+                                       exc: PlayerIsNotLeaderError) -> JSONResponse:
     return JSONResponse(content={
         "message": f"player {exc.player_name} is not a room leader"
     },
```

Comment:
This is an interesting approach to error handlers that I would like to record somewhere.

(Since, there is no written convention, make this comment a thread)

Reply:
what would you like to inquier about this method?

Reply:
Simply address error handlers with proper conventions